rspec-mode.el

```diff
@@ -635,6 +651,15 @@ file if it exists, or sensible defaults otherwise."
          (substring file (length remote))
        file))))
 
+(defun rspec--vagrant-wrapper (command)
+  (if (rspec-vagrant-p)
+      (format "vagrant ssh -c 'cd %s; %s'"
+              rspec-vagrant-cwd
+              (replace-regexp-in-string (rspec-project-root)
+                                        rspec-vagrant-cwd
```

Comment:
No, you're right, my suggestion seems rather complex to implement, for little gain: from what I can see, `rspec-vagrant-p` is fast enough for it to be called 10s of times, and the user still won't notice the difference.

So please just double-check that the current code works fine for you, and I'll be happy to merge it.